api/renewer.go

```diff
@@ -184,6 +178,9 @@ func (r *Renewer) renewAuth() error {
 		return ErrRenewerNotRenewable
 	}
 
+	priorDuration := time.Duration(r.secret.LeaseDuration) * time.Second
```

Comment:
Should this be `r.secret.Auth.LeaseDuration` instead of `r.secret.LeaseDuration`?

Reply:
Yes! Good catch.